electron-app/src/components/settings/api-keys/ExchangeSettings.vue

```diff
@@ -99,16 +105,22 @@ import BaseExternalLink from '@/components/base/BaseExternalLink.vue';
 import ConfirmDialog from '@/components/dialogs/ConfirmDialog.vue';
 import MessageDialog from '@/components/dialogs/MessageDialog.vue';
 import ExchangeBadge from '@/components/ExchangeBadge.vue';
+import { convertToGeneralSettings } from '@/data/converters';
 import { exchanges } from '@/data/defaults';
 import { Message } from '@/store/store';
+import { GeneralSettings } from '@/typing/types';
 
-const { mapState } = createNamespacedHelpers('balances');
+const { mapState } = createNamespacedHelpers('session');
```

Comment:
If you do the override here, then mapping the connected exchanges will fail
instead you add a second entry
```typescript
const { mapSessionState } = createNamespacedHelpers('session');
```

Reply:
if you go with the suggested getter approach though it should be `mapGetters` instead

Reply:
done